sortthread.go

```diff
@@ -34,3 +41,26 @@ func formatSortCriteria(criteria []SortCriterion) interface{} {
 	}
 	return fields
 }
+
+type Thread struct {
+	Id       uint32
+	Children []*Thread
+}
+
+func (t *Thread) String() string {
+	return fmt.Sprintf("(%s)", t.to_str())
+}
+
+func (t *Thread) to_str() string {
+	s := strconv.FormatUint(uint64(t.Id), 10)
+	if len(t.Children) > 1 {
+		for _, child := range t.Children {
+			s += fmt.Sprintf(" (%s)", child.to_str())
+		}
+	} else {
+		for _, child := range t.Children {
```

Comment:
No need for a `for` loop

Reply:
If I don't use the for loop and print `t.Children` it will print out raw interfaces.

Reply:
You can check for `len(t.Children) == 1`, and then just use `t.Children[0]`